package academy.devdojo.javacore.Oexceptions.runtime.test.error;

public class RuntimeExceptionTest1 {
    public static void main(String[] args) {
        int[] array={0,1};
        System.out.println(array[2]);
    }
}
